docs/guides/hybrid-models.md

```diff
@@ -84,7 +103,7 @@ For more information about serving FHE models, see the [client/server section](c
 
 ## Client Side
 
-A client application that deploys a model with hybrid deployment can be developed in a very similar manner to on-premise deployment: the model is loaded normally with PyTorch, but an extra step is required to specify the remote endpoint and the model parts that are to be executed remotely.
+To develop a client application that deploys a model with hybrid deployment, follow a process similar to on-premise deployment. The model is loaded normally with PyTorch, but you need to specify the remote endpoint and the model parts to be executed remotely.
```

Comment:
I prefer the old version:
A client application that deploys a model with hybrid deployment can be developed in a very similar manner to on-premise deployment: the model is loaded normally with PyTorch, but an extra step is required to specify the remote endpoint and  the model parts to be executed remotely.

Reply:
In general, we should avoid using past tense in technical documentation. Using active tense and verbs makes sentences easier to read. (See the [style guide](https://developers.google.com/style/voice))

How about
You can develop a client application that deploys a model with hybrid deployment in a very similar manner to on-premise deployment:  Use PyTorch to load the model normally, but specify the remote endpoint and the part of the model to be executed remotely.

Reply:
I like it